topic0KeyPoint1/src/oop1_7/Date.java

```diff
@@ -0,0 +1,45 @@
+package oop1_7;
+
+public class Date {
```

Comment:
this class is OK, but there are other classes in the JDK to use a date like https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html